Week2/homework/js-exercises/4-the-reading-list.js

```diff
@@ -0,0 +1,33 @@
+{
+    'use strict';
+
+    let library = [
+        { title: "Harry Potter",
+        author: "J.K. Rowling",
+        alreadyRead: true,
+        },
+
+        { title: "The Lord of the Rings",
+        author: "J.R.R Tolkien",
+        alreadyRead: false,
+        },
+
+        { title: "Les Misérables",
+        author: "Victor Hugo",
+        alreadyRead: true,
+        },
```

Comment:
👍 for implementation but there seems to be indentation problems here. Please make sure your code looks as beautiful and easy to read as its underlying logic. It's very important.